.github/workflows/performance-benchmarks.yml

```diff
@@ -39,15 +33,6 @@ jobs:
           runTests: false
           working-directory: e2e/cypress
 
-      - name: Wait for PR branch to be built
-        uses: lewagon/wait-on-check-action@v1.2.0
-        with:
-          ref: ${{ github.event.pull_request.head.sha || github.sha }}
-          check-name: untagged-build
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          wait-interval: 10
-          verbose: true
-
       - name: Download the thing
```

Comment:
Could we actually upload the server bundle in the new `prepare-docker-build` job as an artifact and then redownload it here instead of having to recreate the server bundle here?

Reply:
This is an effort, to replicate the functionality of CircleCI to GitHub Actions. This is currently the logic implemented of the process in CircleCI. 
If there is need to improve the logic/workflow , I'd suggest we discuss this over a second iteration.

Let me know @hmhealey